browser/brave_content_browser_client.cc

```diff
@@ -173,7 +173,7 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "third_party/blink/public/mojom/loader/resource_load_info.mojom-shared.h"
 #endif
 
-#if BUILDFLAG(ENABLE_WIDEVINE)
+#if BUILDFLAG(ENABLE_WIDEVINE) && !BUILDFLAG(IS_ANDROID)
 #include "brave/browser/brave_drm_tab_helper.h"
```

Comment:
ah sorry, my bad, I saw it's as included for some reason :).

Reply:
@SergeyZhukovsky we still need the same opt-in functionality here because we shouldn't be playing drm videos without user permission. cc @pes10k

Reply:
The permission is not just about downloading widevine support on desktop, it's about getting user permission before playing widevine because it's using closed source code with potential user tracking, etc...

Reply:
unless I'm misunderstanding something and that permission prompt for first time widevine access happens somewhere else

Reply:
if this code isn't triggered on android because it plays widevine through an OS sdk then we need an alternate way to get user permission before the first time we play widevine

Reply:
@bridiver do you have an example of such a user permission? Yes it's fully from Android OS SDK.

Reply:
In my mind the user permission is their login and password for websites where the DRM content is served.

Reply:
The user logging into a site is not at all comparable to a permission. This is concerning.  Are there other places in Android treating a login as permission for advanced/restricted capabilities?

Reply:
@SergeyZhukovsky this is what we do on desktop. We have a one-time permission dialog for widevine. Also I don't believe that all widevine content is necessarily behind a user login. I'm not asking for new functionality here, I just want to make sure we have the same one-time permission here that we do on desktop.

Reply:
> Are there other places in Android treating a login as permission for advanced/restricted capabilities?

@pes10k we don't do anything different related to permissions on Android. It's just that feature as it uses Android OS SDK, that's why a permission about downloading and using a third party code has not been done on Android.